build/codegen/codegen.sh

```diff
@@ -22,6 +22,11 @@ codegendir="${scriptdir}/../../vendor/k8s.io/code-generator"
 # ensures the vendor dir has the right deps, e,g. code-generator
 go mod vendor
 
+# Set these so that we don't 'go get' cross-compiled binaries
+GOOS="$(go env GOHOSTOS)"
+GOARCH="$(go env GOHOSTARCH)"
+export CGO_ENABLED=0
```

Comment:
Allow `make codegen` on mac.

Reply:
Separate PR needed? though I'm able to run `make codegen` on mac without this...

Reply:
+1 I don't need that to run codegen.